src/main/java/shop/service/impl/UserServiceImpl.java

```diff
@@ -34,6 +37,9 @@ public UserResponseDto register(UserRegistrationRequestDto registerUserRequestDt
         user.setRoles(Set.of(roleRepository.findByRole(Role.RoleName.ROLE_USER)
                 .orElseThrow(() -> new EntityNotFoundException("Can`t find role"))));
         userRepository.save(user);
+        ShoppingCart shoppingCart = new ShoppingCart();
+        shoppingCart.setUser(user);
+        shoppingCartRepository.save(shoppingCart);
```

Comment:
create and implement `createShoppingCart` method in ShopCartService

```suggestion
        shoppingCartService.createShoppingCart(user);
```